src/ui/backup.ts

```diff
@@ -19,7 +19,7 @@ async function backup(_ui: UI) {
             _ui.instance.alert(_ui.instance.i18n.updateSuccess);
           } else if (localStorage.dropboxRevoked === 'true') {
             _ui.instance.alert(String(_ui.instance.i18n.token_revoked)
-                                   .replace(/\${service}/, 'Dropbox'));
+                                   .replace(/\$?{?service}/, 'Dropbox'));
```

Comment:
It seems not make sense. `/\$?{?service}/` matches `${service}`, `{service}`, `$service}` and `service}`

Reply:
The most I can do to make it more specific is this: `/(?:\${)?service}/`.

Why:
Firefox
![image](https://user-images.githubusercontent.com/27789806/49688959-6f7e4f80-fadf-11e8-8b4e-308e8382b90e.png)
Chrome
![image](https://user-images.githubusercontent.com/27789806/49688966-8c1a8780-fadf-11e8-99b0-0ca159a1ac2a.png)

Reply:
`${variable}` is parsed as variable reference in JS. `{{replacement}}` would be better for template use.

Reply:
It seem i18n message supports replacement. See https://developer.chrome.com/extensions/i18n#examples-getMessage and https://chromium.googlesource.com/chromium/src/+/master/chrome/common/extensions/docs/examples/extensions/news/_locales/en/messages.json#73